src/main/kotlin/domain/WinningNumber.kt

```diff
@@ -0,0 +1,18 @@
+package domain
+
+class WinningNumber(private val lottoNumbers: List<LottoNumber>, private val bonusNumber: LottoNumber) :
+    Lotto(lottoNumbers) {
```

Comment:
```suggestion
class WinningNumber(
    private val lottoNumbers: List<LottoNumber>, 
    private val bonusNumber: LottoNumber
) : Lotto(lottoNumbers) {
```
가로로 너무 길어질 경우에는 개행을 하는 것을 권장해요.

ref. https://kotlinlang.org/docs/coding-conventions.html#class-headers